src/main/java/com/github/wovnio/wovnjava/UrlLanguagePatternHandler.java

```diff
@@ -10,6 +10,8 @@
 
     abstract String insertLang(String url, String lang);
 
+    abstract boolean isMatchSitePrefixPath(String url);
```

Comment:
I would not make it abstract and implement default returning `true`. That would avoid having to think about it when implementing URL handler for non-path URL components. Only concerned handlers would override it.

Both solution are fine. I prefer the one I mentioned so I wanted to bring that up.

Reply:
That's nicer